druid/src/test/java/org/apache/calcite/test/DruidAdapterIT.java

```diff
@@ -164,7 +164,7 @@ public Void apply(List list) {
         + "filter=[=($17, 'Jeremy Corbyn')], groups=[{7}], aggs=[[]])\n";
     final String druidQuery = "{'queryType':'groupBy',"
         + "'dataSource':'wikiticker','granularity':'all',"
-        + "'dimensions':['countryName'],'limitSpec':{'type':'default'},"
+        + "'dimensions':[{\"type\":\"default\",\"dimension\":\"countryName\"}],'limitSpec':{'type':'default'},"
```

Comment:
Why are we getting escape + " instead of single quote? Is it possible to keep it the same?